bibliopixel/led.py

```diff
@@ -1,4 +1,6 @@
-import math, threading, time
+import math
+import threading
+import time
```

Comment:
lol... hey, it's not me, it's my linter :P That's the coding standard I have at work, though I don't entirely like it. I think i'll disable that in the flake8 config.